.github/workflows/release.yml

```diff
@@ -11,75 +13,77 @@ jobs:
-    runs-on: ubuntu-latest
-    permissions:
-      id-token: write
-    needs:
-      - ci
-    steps:
-    - uses: actions/checkout@v2
-    - uses: sigstore/cosign-installer@main
-    - name: Setup rust toolchain
-      uses: actions-rs/toolchain@v1
-      with:
-        toolchain: stable
-    - name: Setup musl for aarch64
-      run: |
-        curl https://musl.cc/aarch64-linux-musl-cross.tgz | tar -xz
-        echo "$PWD/aarch64-linux-musl-cross/bin" >> $GITHUB_PATH
-    - run: rustup target add aarch64-unknown-linux-musl
-    - name: Build kwctl
-      env:
-        CC: aarch64-linux-musl-gcc
-      run: cargo build --target=aarch64-unknown-linux-musl --release
-    - run: mv target/aarch64-unknown-linux-musl/release/kwctl kwctl-linux-aarch64
-    - name: Sign kwctl
-      run: cosign sign-blob kwctl-linux-aarch64 --output-certificate kwctl-linux-aarch64.pem --output-signature kwctl-linux-aarch64.sig
-      env:
-        COSIGN_EXPERIMENTAL: 1
-    - run: zip -j9 kwctl-linux-aarch64.zip kwctl-linux-aarch64 kwctl-linux-aarch64.sig kwctl-linux-aarch64.pem
-    - name: Upload binary
-      uses: actions/upload-artifact@v2
-      with:
-        name: kwctl-linux-aarch64
-        path: kwctl-linux-aarch64.zip
+          sudo sed -i "s/deb h/deb [arch=amd64] h/g" /etc/apt/sources.list
+
+          sudo sed -i "$ a deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy main restricted" /etc/apt/sources.list
+          sudo sed -i "$ a deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy-updates main restricted" /etc/apt/sources.list
+          sudo sed -i "$ a deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy universe" /etc/apt/sources.list
+          sudo sed -i "$ a deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy-updates universe" /etc/apt/sources.list
+          sudo sed -i "$ a deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy multiverse" /etc/apt/sources.list
+          sudo sed -i "$ a deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy-updates multiverse" /etc/apt/sources.list
+          sudo sed -i "$ a deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy-backports main restricted universe multiverse" /etc/apt/sources.list
+          sudo sed -i "$ a deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy-security main restricted" /etc/apt/sources.list
+          sudo sed -i "$ a deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy-security universe" /etc/apt/sources.list
+          sudo sed -i "$ a deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy-security multiverse" /etc/apt/sources.list
+
+          sudo apt update -y
+      - name: Install Musl and configure gcc spec
+        run: |
+          sudo apt install -y musl-dev:${{ matrix.arch }}
+          sudo sed -i 1d /usr/lib/${{ matrix.targetarch }}-linux-musl/musl-gcc.specs
+          sudo sed -i "s/*cpp_options/+cpp_options/g" /usr/lib/${{ matrix.targetarch }}-linux-musl/musl-gcc.specs
```

Comment:
I would like to add a comment before the 2 lines patching the spec. Something like:
`# patching the .spec file, as by default it has a bug where it tries to set old_cpp_options but it already exists. using *+cpp_options achieves the same desired functionality of appending preexisting options`